### PR TITLE
Use :loose option with feed-normalizer for now

### PR DESCRIPTION
**Pull request by jermie**
Using :loose allows capturing multiple categories, while only falling back to SimpleRSS when necessary. 

https://github.com/aasmith/feed-normalizer/blob/master/lib/feed-normalizer.rb#L105

### CODE REVIEW

**Comment by cantino** [comment]
Thanks @jermie!

@knu, this look good to you?

**Comment by knu** [comment]
You should write a spec for this, using a feed containing items with multiple categories.

Currently RssAgent does not have enough specs, and that makes it hard to modify it without breaking compatibilities.  We'll have to add more and more specs from now on.

**Comment by knu** [comment]
Also, I prefer the modern hash syntax (`loose: true`) for new code.

**Comment by cantino** [comment]
If you need a hand with the spec, please let me know.  I'm glad to help. Do
you have an example feed that this change helps with?

**Comment by jermie** [comment]
@cantino Thanks! I'll get a spec for this together and let you know if I run into trouble. Here's an example feed of where this helps: http://onethingwell.org/rss

Without :loose only one category gets captured.

**Comment by jermie** [comment]
@cantino @knu how does this look now?

File: spec/models/agents/rss_agent_spec.rb [hunk 1]
```diff
@@ -149,9 +150,23 @@
...
 
+  context "parsing feeds" do
+    before do
+      @valid_options['url'] = 'http://onethingwell.org/rss'
+    end
+
+    it "captures multiple categories" do
+      agent.check
+      first, second, third = agent.events.take(3)
+      expect(first.payload['categories'].count).to eq(3)
```
Comment (cantino) [inline spec/models/agents/rss_agent_spec.rb]: Maybe add a line asserting what the categories are?

**Comment by cantino** [comment]
Thanks for updating this, it looks great!

File: spec/models/agents/rss_agent_spec.rb (hunk 1 above)
Comment (jermie) [inline spec/models/agents/rss_agent_spec.rb]: :+1:

**Comment by cantino** [comment]
Thanks @jermie!

**Pull request merged**